index.js

```diff
@@ -107,6 +108,11 @@ module.exports = {
           delete req.session.persona;
         }
 
+        if (req.query.persona === 'none') {
```

Comment:
I don't think this code is reachable, you already redirected and now you're looking at the query again. Maybe you meant to look at `req.session.nextPersona`.